doc/components.md

```diff
@@ -422,6 +422,99 @@ It is also possible to create custom renderers by extending the `ParallaxRendere
 Three example implementations can be found in the
 [examples directory](https://github.com/flame-engine/flame/tree/main/examples/lib/stories/parallax).
 
+## ShapeComponents
+
+There exist three basic components that can be used if you want to draw geometrical shapes as
+components on the screen. Since the `ShapeComponent`s are `PositionComponent`s you can use effects
```

Comment:
I think you should first say what ShapeComponent represents abstractly, and then mention there are 3 provided implementations ?

Reply:
Good call, fixed.